lib/polyn/configuration.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative "naming"
```

Comment:
I think we've tried to steer clear of require_relative as a rule. I think in spiff_rb we've opted to just eager load everything up front instead.

Reply:
oh interesting, I don't know much about that. Where does the eager loading happen? What problems does `require_relative` cause?

Reply:
I'm less familiar with how Ruby loads it, so someone else will have to help with that. I just know that we got into a situation where it would get caught in a load loop and die...

Reply:
oh no! that sounds bad. I'll look into that more

Reply:
Rails uses its own Autoloader to locate files. I assumed that being in the `lib` directory maybe sidestepped the loader and that's why all this `require` stuff was put in there, but this comment thread makes me wonder if that assumption was invalid. 😅